github_pr.py

```diff
@@ -124,6 +131,16 @@ def github_comment_pr(**args):
     pr.create_issue_comment(args['body'])
 
 
+def github_list_pr_comments(**args):
```

Comment:
what calls this fn? anything? don't we get the comments plus all the other stuff we need (`comment.author`) by calling `github_list_prs` with `args['comments']`?